nautobot/cloud/templates/cloud/cloudservice_retrieve.html

```diff
@@ -43,3 +39,13 @@
         </div>
     </div>
 {% endblock content_right_page %}
+
+{% block content_full_width_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Cloud Networks</strong>
+        </div>
+        {% include 'inc/table.html' with table=networks_table %}
+    </div>
+    {% include 'inc/paginator.html' with paginator=networks_table.paginator page=networks_table.page %}
+{% endblock content_full_width_page %}
```

Comment:
1) Should this be implemented as a separate tab rather than a panel on the main tab?

2) Should we also add a `services_table` to `cloudnetwork_retrieve.html`?

Reply:
Cloud network has `prefixes` and `circuits` on the main tab. If we're not creating a separate view, the performance impact is the same either way so it's probably better to have in a separate tab. Shall we move the prefixes and circuits for CloudNetwork to a separate tab too?

Reply:
Cloud resource types has `services` and `networks` in separate tabs btw

Reply:
I went ahead and added the tabs to CloudNetwork